### ISSUE TITLE
Cannot get drag to reorder to work

### ISSUE THREAD

**Issue by allanlundhansen** [JackAdams/meteor-editable-list#6]
For some reason this has stopped working locally but seems to be the case also for the demo app http://editable-text-demo.meteor.com/

**Comment by JackAdams** [comment]
Yes, it used to have a dependency on `mizzao:jquery-ui`, which provided the sortability.  I didn't want to pull in the whole of jquery-ui just for that little feature every time, so changed it in [this commit](https://github.com/JackAdams/meteor-editable-list/commit/3848f7de6211ac9c32aa3e9527f86ac3e0440b4e), so that the sortable lists only work if jquery-ui is added.

For some reason, if I remember correctly, the only jquery-ui package that seems to work is `mrt:jquery-ui` which is probably horribly out of date.

Also, the sortable code had a pretty filthy hack in it to overcome a conflict between jquery-ui and blaze.

For these reasons, I've left the sortable functionality undocumented.

I'm wide to open to accepting a PR for the package that restores sortability, provided it can be done with a lighter library than the whole of jquery-ui.  I think someone published jquery-ui's drag and drop functionality as a standalone package, but haven't had time to test whether that includes sortable. `rubaxa:sortable` is another package that may be lighter than jquery-ui but, again, I haven't had time to experiment with it.

**Comment by allanlundhansen** [comment]
Ok thank you for your quick reply, I have "hack" fixed it for now with https://github.com/mquandalle/meteor-jquery-ui-drag-drop-sort

**Comment by JackAdams** [comment]
Any way the hack could be easily worked into the package to have sortable lists by default? Will totally accept a PR. :-)
